ROADMAP.md

```diff
@@ -0,0 +1,43 @@
+# Roadmap for v2
+
+Over the years, there have been many great suggestions that we were not able to implement due to them being a breaking change.
+Now, we are currently work on Prism v2.
+
+## ESM
```

Comment:
```suggestion
## ESM for component dependencies
```

Reply:
Not really true. Everything will use ESM.